stock_mts_mto_mrp_rule/tests/test_stock_move.py

```diff
@@ -0,0 +1,90 @@
+from odoo.tests import TransactionCase
+
+
+class TestStockMove(TransactionCase):
+    def setUp(self):
```

Comment:
Prefer using setupClass as it would be instantiated just once between all tests.

Reply:
Resolved